plugin/clone/src/clone_client.cc

```diff
@@ -1036,6 +1036,9 @@ bool Client::plugin_is_installed(std::string &plugin_name) {
 
 int Client::validate_remote_params() {
   int last_error = 0;
+  bool donar_rocksdb_installed = false;
+
+  std::string ROCKSDB = "ROCKSDB";
```

Comment:
```suggestion
  static constexpr char ROCKSDB[] = "ROCKSDB";
```

This creates a new string every time this method is called. `static` would avoid that, but `constexpr` would move it to compile-time. As C++17 does not yet have a constexpr std::string, a char array would do.